source/upgrading/upgrade/_sysvm_restart.rst

```diff
@@ -15,7 +15,22 @@
 
 .. sub-section included in upgrade notes.
 
-Once you've upgraded the packages on your management servers, you'll
+.. note::
+
+   From Apache CloudStack version 4.17.0 onward, there is support to live patch 
+   systemVMs, namely, SSVM, CPVM, Routers. Live patching provides support 
+   for zero-downtime upgrades, wherein, the SystemVM software is updated to the
+   latest code version without having to destroy and recreate them / restart them.
+
+   With this feature, users will have a choice wherein they can use the existing systemVM template with the latest
+   software by using the live patch feature, or can follow the usual workflow of restarting the
+   systemVM to use the latest systemVM template. Live Patching systemVMs serves to be especially
+   useful in cases when the code version has upgraded but the template hasn't. In such a scenario users
+   will no longer need to restart the systemVMs to use the latest code.
```

Comment:
@Pearl1594 can you explain in some more detail what is covered in the "zero downtime" and what isn't, esp in the context of CPVM and VRs.